aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Assets.Scripts.Tiles;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Assets.Scripts.WFC
+{
+    /// <summary>
+    /// Represents the grid placer.
+    /// </summary>
+    public class GridPlacer : MonoBehaviour
+    {
+        /// <summary>
+        /// Represents the grid.
+        /// </summary>
+        private Grid _grid;
+
+        /// <summary>
+        /// Represents the room objects.
+        /// </summary>
+        public RoomObjects RoomObjects;
+
+        /// <summary>
+        /// Awake is called when the script instance is being loaded.
+        /// </summary>
+        public void Awake()
+        {
+            _grid = new Grid(5, 5);
+
+            _grid.Init();
+            
+            TempFillFunction();
+
+            for (int y = 0; y < _grid.Height; y++)
+            {
+                for (int x = 0; x < _grid.Width; x++)
+                {
+                    PlaceTile(x, y, _grid[x, y].Tile);
+                }
+            }
+        }
+
+        /// <summary>
+        /// A temporary function to fill the grid with rooms.
+        /// </summary>
+        public void TempFillFunction()
+        {
+            _grid.PlaceRoom(2, 1, new Room(new List<bool> { false, true, true, false }));
+            
+            _grid.PlaceRoom(4, 4, new Room(new List<bool> { false, true, true, false }));
+
+            // Road 1
+            _grid[2, 2].Tile = new Straight();
+            _grid[2, 3].Tile = new Straight();
+            _grid[2, 4].Tile = new Corner(2);
+            _grid[3, 4].Tile = new Straight(1);
+            
+            // Road 2
+            _grid[3, 1].Tile = new Straight(1);
+            _grid[4, 1].Tile = new TSection(3);
+            _grid[4, 2].Tile = new Straight();
+            _grid[4, 3].Tile = new Straight();
+            _grid[4, 0].Tile = new DeadEnd();
+        }
```

Comment:
By temporary, I think you mean this will be removed when WFC is implemented?

Reply:
Yupp